source/indexes/atlas-search-index.txt

```diff
@@ -0,0 +1,274 @@
+.. _cpp-atlas-search-index:
+
+====================
+Atlas Search Indexes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: index, query, text search, efficiency
+
+Overview
+--------
+
+The MongoDB Atlas Search feature enables you to perform full-text
+searches on collections hosted on Atlas. Before you can perform Atlas
+Search queries, you must create indexes that specify which
+fields to index and how they are indexed.
+
+To learn more about Atlas Search, see the :atlas:`Atlas Search Overview
+</atlas-search/atlas-search-overview/>`.
+
+You can use the following methods on a ``mongocxx::search_index_view`` instance
+to manage your Atlas Search indexes:
+
+- ``create_one()``
+- ``create_many()``
+- ``list()``
+- ``update_one()``
+- ``drop_one()``
+
+To get a ``search_index_view`` to your collection, 
+call the ``search_indexes()`` method on your ``mongocxx::collection`` instance.
```

Comment:
S: I'm not sure what a search_index_view is, and maybe for that reason, this sentence feels a little tacked on. It seems like this should be up before the "You can use the following methods..." paragraph, then you can segue into the methods available on it.

Reply:
Your comment made me realize that I never introduced how to connect to the database/collection, so I added a [Sample Data](https://deploy-preview-64--docs-cpp.netlify.app/indexes/atlas-search-index/#sample-data) section similar to what the [Compound Indexes](https://deploy-preview-64--docs-cpp.netlify.app/indexes/compound-index/) / [Single Field Indexes](https://deploy-preview-64--docs-cpp.netlify.app/indexes/single-field-index/) pages have. This introduces how to connect to the database and also how to instantiate a ``search_index_view``. This allowed me to remove the line `auto siv = collection.search_indexes();` from every code example and only include it in this beginning section. Thanks for pointing this out! If you think there's a more condensed way to organize the Sample Data section, I'm also open to suggestions!